CMakeLists.txt

```diff
@@ -369,6 +369,7 @@ else()
   set(EXTRA_WARNINGS
     -Wold-style-declaration
     -Wunused-but-set-parameter
+    -Wno-override-init
```

Comment:
I just realized there is a `no-` here. So instead of enabling the warning, this disables the warning. This is the opposite that we want. @vivinperis  Could you please check this again a fix it? I am reopening the referred issue.